### ISSUE TITLE
Rate Limiting Issues With I2P Accesss

### ISSUE THREAD

**Issue by bonedaddy** [RTradeLtd/Temporal#485]
**Describe the bug**
When users access our service via I2P, all access to the API will appear to be coming from the LAN IP address of our I2P router, as opposed to an actual client IP. 

**Expected behavior**

* Dont rate limit I2P users since I2P essentially acts as a "rate limit" anyways. 

I'm not quite sure how to solve this without increasing per-IP rate limits.

**Comment by xiegeo** [comment]
White list LAN IP addresses?

**Comment by bonedaddy** [comment]
Good idea, i think that's possible with the rate limiter we use? I will check
